frontend/src/features/projects/spl/steps/SplStep.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import { Formik, Form } from 'formik';
+import _ from 'lodash';
+import { useState } from 'react';
+import { formatDate } from 'utils';
+import { Container } from 'react-bootstrap';
+import styled from 'styled-components';
+import StepErrorSummary from '../../common/components/StepErrorSummary';
+import { useDispatch, useSelector } from 'react-redux';
+import { RootState } from 'reducers/rootReducer';
+import {
+  useStepForm,
+  StepStatusIcon,
+  SPPApprovalTabs,
+  ReviewWorkflowStatus,
+  IProject,
+  IProjectTask,
+  IStepProps,
+  useProject,
+  handleValidate,
+  ValidationGroup,
+  DisposeWorkflowStatus,
+} from '../../common';
+import {
+  saveSplTab,
+  SplTabs,
+  SurplusPropertyInformationYupSchema,
+  SurplusPropertyListYupSchema,
+} from '..';
+import { ApprovalActions } from 'features/projects/erp';
+import './SplStep.scss';
+import { DocumentationStepSchema } from 'features/projects/dispose';
+
+const CenterBoldText = styled.div`
+  text-align: center;
+  font-family: 'BCSans-Bold';
+  margin-bottom: 1.5rem;
+`;
+
+const copyAppraisalTasks = (project: IProject): IProject => {
```

Comment:
Unfortunately we need to duplicate any interactions the user may have had with the the optional appraisal tasks to the required appraisal tasks displayed on this form.